CHANGES/9586.bugfix

```diff
@@ -0,0 +1 @@
+Container clients fail to retrieve artefacts from S3 storage
```

Comment:
Fixed a bug that caused container clients to be unable to interact with content stored on S3.